extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -187,6 +198,8 @@ export class PublishDatabaseDialog {
 					width: '100%'
 				});
 
+			this.formBuilder.insertFormItem(publishProfileFormComponentGroup, 2);
```

Comment:
can this be added with the other form items above?

Reply:
I added it to the form item above, but then the Profile title isn't bold and doesn't really look like a title. Let me know if it looks ok, or if it should be updated.